### PR TITLE
Improve Accessibility for The Portfolio

### PR DESCRIPTION
**Pull request by mhamedGd**
- Page Title - Changed to "Microverse Portfolio (second template)".
- Image Alternatives - Added image alternatives to all images in the html document.
- Text headings - No issue found.
- Color contrast - Followed template guides.
- Resize - No issue found.
- Interaction - Added label tags to the contact form starting from line (324).
- Moving Content - There are no moving content in the project.
- Multimedia - No multimedia beside the images.
- The basic structure of the page - No issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;600;700&display=swap" rel="stylesheet">
   <script src="src/script/index.js" defer></script>
-  <title>Portfolio</title>
+  <title>Microverse Portfolio (second template)</title>
```
Comment (isaacpitwa) [inline index.html:13]: - [ ] Great job updating the webpage title. i believe there is room for improvement. by making the title more specific and not including **Microverse** in the title for example.  **Mark's portifolio** , **Mark full-stack webdeveloper**

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi Team 👋🏽 🙂 ,

You have done a tremendous job setting up your project. I really like it :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
-  Clean and readable code base ✅

## Required changes ♻️ 

Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (arabiu033) [reply index.html]: Thank you. adjusted.

**Review by zhadier — APPROVED** [review]
## Hi @mhamedGd  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

- [x] From now onwards, kindly ensure that in pair-programming projects there are commits from all team members in the repo during the first submission. Not doing so will result in an invalidation in most cases.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**